test/functional/rpc_psbt.py

```diff
@@ -168,6 +168,9 @@ def run_test(self):
             assert tx_in["sequence"] > MAX_BIP125_RBF_SEQUENCE
         assert_equal(decoded_psbt["tx"]["locktime"], 0)
 
+        # Make sure change address wallet does not have P2SH innerscript access to results in success
```

Comment:
nit: typo in comment

Reply:
not fixed. Suggest something like "Make sure that the wallet can successfully create a funded psbt when it isn't able to sign for the change output" or similar.